lib/persist.js

```diff
@@ -1,50 +1,66 @@
 require("dotenv").config();
-const redis = require("redis").createClient(process.env.REDIS_URL);
-const { promisify } = require("util");
-const get = promisify(redis.get).bind(redis); // this maps redis.get(key, callback(results)) to a promise-compatible function
+
 const moment = require("moment");
 const { getCurrentEmojis } = require("./slack");
 
+const makeRedisClient = async () => {
```

Comment:
TODO: probably need to `disconnect` these clients at some point instead of just letting them timeout.